src/repositories/inventory.js

```diff
@@ -1,15 +1,20 @@
 import api from "@/api";
 
+const state = {
+  products: []
+};
```

Comment:
Ici, on est obligé de faire ça, parce que `this.state` est undefined lorsque `getProductsCount()` et donné a Vue dans un computed.